pkg/cmd/pipeline/translate_test.go

```diff
@@ -91,6 +93,9 @@ func Test_AddDevAnnotations(t *testing.T) {
 	manifest := &model.Manifest{
 		Namespace: "namespace",
 		Dev: model.ManifestDevs{
+			"not-found-deployment": &model.Dev{
+				Name: "not-found-deployment",
+			},
```

Comment:
Just curiosity, Why this change?

Reply:
That tests an untested scenario which checks what happens if the deployment is no longer running. We should skip the adding annotation function for that resource